base/intset.jl

```diff
@@ -220,6 +220,12 @@ function ==(s1::IntSet, s2::IntSet)
     return true
 end
 
+"""
+    issubset(A, S) -> Bool
+    ⊆(A,S) -> Bool
+
+Return `true` if `A` is a subset of or equal to `S`.
```

Comment:
I didn't think `IntSet` was very widely used, but it shows up in more packages than I thought it might. The only package where both `IntSet` and `issubset` appear in the same file is DataStructures.jl which defines its own `IntSet`, if that's worth anything. I think the result should be equivalent and `IntSet` happens not to use `in` just as an optimization, and any user-defined type could choose to do the same with its own specialization of `issubset`. So I lean towards getting rid of the `IntSet` specific version of the docstring, but could be convinced otherwise if someone thinks it's important to keep.